@chakra-ui/cli ypeError: mem_1.default is not a function

Hi,

I'm running into some issues when trying to use the CLI.

Here is the output I have
```
> npx chakra-cli tokens ./libs/theme/src/lib/theme.ts

 Chakra UI CLI  v1.3.1 by chakra UI
Generate theme typings for autocomplete


✖ An error occurred
TypeError: mem_1.default is not a function
```

CLI version: 1.3.1
OS: OSX
Node: 14.15.4
Npm: 6.14.10

Tip: Downgrading to v 1.2.0 give me same error but with a different message:

```
⨯ Unable to compile TypeScript:
../libs/theme/src/lib/colors.ts(4,8): error TS1259: Module '"xxxxx/node_modules/mem/dist/index"' can only be default-imported using the 'esModuleInterop' flag
```

--- a/.changeset/lemon-deers-carry.md
+++ b/.changeset/lemon-deers-carry.md
@@ -1,0 +1,5 @@
+---
+"@chakra-ui/cli": minor
+---
+
+Enable esModuleInterop for `chakra-cli tokens`

--- a/tooling/cli/bin/tsconfig.json
+++ b/tooling/cli/bin/tsconfig.json
@@ -4,7 +4,8 @@
     "module": "commonjs",
     "allowJs": true,
     "strict": false,
-    "noImplicitAny": false
+    "noImplicitAny": false,
+    "esModuleInterop": true
   },
   "ts-node": {
     "transpileOnly": true

